.travis.yml

```diff
@@ -73,8 +73,10 @@ matrix:
         - os: osx
           stage: Cron tests
           env: SETUP_CMD='test --remote-data=astropy'
-               CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PIP_DEPENDENCIES='jplephem' EVENT_TYPE='cron'
+               CONDA_DEPENDENCIES="$CONDA_ALL_DEPENDENCIES clang llvm-openmp"
+               PIP_DEPENDENCIES='jplephem'
+               CCOMPILER=clang
```

Comment:
Where is this variable used?

Reply:
Hmm, good catch. I was completely unaware of https://github.com/astropy/astropy-helpers/pull/395. Would have been helpful to have been tagged in that, just for my awareness.

Anyhow, I'll take a better look and fix it. Thanks for spotting this.